### PR TITLE
Convert to target .NET Core and .NET 4.5

### PR DESCRIPTION
**Pull request by neoeinstein**

### CODE REVIEW

File: .vscode/extensions.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (wallymathieu) [inline .vscode/extensions.json:1]: should these be included in the source?

File: src/Hekate/Hekate.fsproj [hunk 2]
```diff
@@ -1,91 +1,17 @@
...
-      </PropertyGroup>
-    </When>
-    <Otherwise>
-      <PropertyGroup Condition="Exists('$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)\FSharp\Microsoft.FSharp.Targets')">
-        <FSharpTargetsPath>$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)\FSharp\Microsoft.FSharp.Targets</FSharpTargetsPath>
-      </PropertyGroup>
-    </Otherwise>
-  </Choose>
-  <Import Project="$(FSharpTargetsPath)" />
+  <Import Project="..\nuget.props" />
```
Comment (wallymathieu) [inline src/Hekate/Hekate.fsproj]: this should be ..\\..\nuget.props

File: tests/Hekate.Tests/Hekate.Tests.fsproj [hunk 3]
```diff
@@ -1,152 +1,30 @@
...
-    <ProjectGuid>e8065e21-b677-46c1-9af5-eb2195f83c23</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <RootNamespace>Hekate.Tests</RootNamespace>
-    <AssemblyName>Hekate.Tests</AssemblyName>
-    <TargetFrameworkVersion>v4.5.2</TargetFrameworkVersion>
-    <TargetFSharpCoreVersion>4.4.0.0</TargetFSharpCoreVersion>
-    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
-    <Name>Hekate.Tests</Name>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
```
Comment (wallymathieu) [inline tests/Hekate.Tests/Hekate.Tests.fsproj]: Since fscheck does not support netcoreapp1.1 you could use net452 and downgrade Unquote

**Comment by wallymathieu** [comment]
I've added a pull request to your pull request ;) https://github.com/neoeinstein/hekate/pull/1

**Comment by neoeinstein** [comment]
@wallymathieu I produced some temporary packages for FsCheck and Unquote. They are hosted on my nuget feed at the moment. I wanted to be sure that everything worked properly on netcore. Downgrading to net452 is a fine idea until these packages are upgraded too.

**Comment by wallymathieu** [comment]
I was hoping that this pull request could be merged soon as I want to write some graph algorithms in core.

**Comment by neoeinstein** [comment]
Will do. I'm away from my computer at the moment. I will work on merging the several outstanding xyncro PRs, including this one, today. Pushed NuGet packages should follow shortly after that.

**Comment by wallymathieu** [comment]
Sweet! 😄

**Comment by neoeinstein** [comment]
Updated Aether at 8.2.0 has been published. Now ready to merge.

**Pull request merged**